File: CodeIQ-q969/CodeIQ-q969-1.pas
var i:integer;
for i:=1 to 50 do
begin
if i mod 15=0 then
write("FIZZBUZZ")
else if i mod 3=0 then
write("fizz")
else if i mod 5=0 then
write("buzz")
else
write(i)
end

File: CodeIQ-q969/CodeIQ-q969-1.f
integer i
do i=1, 50
if(mod(i,15)<1)then
write (*,'(a)',advance='no')"FIZZBUZZ"
else if(mod(i,3)<1)then
write (*,'(a)',advance='no')"fizz"
else if(mod(i,5)<1)then
write (*,'(a)',advance='no')"buzz"
else
write(*,'(i0)',advance='no')i
end if
end do